packages/react-native-sdk/src/components/Participant/LocalParticipantView/index.tsx

```diff
@@ -115,7 +135,15 @@ export const LocalParticipantView = ({
             <ParticipantView
               participant={localParticipant}
               videoMode={'video'}
-              style={[styles.floatingViewContainer, style]}
+              style={[
+                styles.floatingViewContainer,
```

Comment:
Can we rename this(and also the theme variable below) style to `participantViewStyle`/`participantViewContainer` since it is applied to the participantView?